scripts/configuration.md

```diff
@@ -16,6 +16,26 @@ unique identifier unit
 
 
 
+## settings.id_characteristics
```

Comment:
Should we call this 'id_characteristics_cols' to this to match standard convention?

Reply:
On the other hand we have `filters` and `details` instead of `filter_cols` and `detail_cols` because they can represent an array of objects with metadata like label/header.

Reply:
hm - you're right. with that in mind, i'm ok leaving this as it is currently written (for now at least)